tests/e2e/Services/Account/AccountBase.php

```diff
@@ -51,41 +51,6 @@ public function testCreateAccount(): array
 
         $this->assertEquals($response['headers']['status-code'], 409);
 
-        $response = $this->client->call(Client::METHOD_POST, '/account', array_merge([
```

Comment:
any reason to remove the tests?

Reply:
Because of the optional params we added, this tests don't need to fail anymore. I'll bring it back if we change the endpoint.

Reply:
Understood 👍

Reply:
Tests re-added